src/common/contexts/TokenInformationContext/TokenInformationContext.tsx

```diff
@@ -6,8 +6,8 @@ import { useProviderContext } from "../provider";
 import { useSupportsInterface } from "../../hooks/useSupportsInterface";
 
 interface TokenInformationContext {
-  tokenRegistryAddress: string;
-  tokenId: string;
+  tokenRegistryAddress: string | undefined;
```

Comment:
Why not using optional parameter like for the rest? :
- token?: string

Reply:
because these are required parameters for the constructor(initialise method) so wanted to type them as required not optional.